apps/web/next.config.mjs

```diff
@@ -1,4 +1,42 @@
+import path from 'path'
+import { fileURLToPath } from 'url'
+
+const currentFileURL = import.meta.url
+const __filename = fileURLToPath(currentFileURL)
+const __dirname = path.dirname(__filename)
+
+const PACKAGES = ['env', 'core', 'jobs']
+const PACKAGE_NAMES = PACKAGES.map((pkg) => `@latitude-data/${pkg}`)
+
+function buildAlias({ localPackages }) {
+  return localPackages.reduce((aliases, pkg) => {
+    aliases[`$${pkg}`] = path.resolve(__dirname, `../../packages/${pkg}/src`)
+    return aliases
+  }, {})
+}
+
 /** @type {import('next').NextConfig} */
-const nextConfig = {};
+const nextConfig = {
+  output: 'standalone',
+  transpilePackages: Object.values(PACKAGE_NAMES),
```

Comment:
Also, these would be only for development. We're not ready for real production so I'm happy with this for now.